src/languages/en.ts

```diff
@@ -2416,8 +2417,77 @@ export default {
+                        helpText: ' on configuring custom segements/records.',
+                        emptyTitle: 'Add a custom segment or custom record',
+                        fields: {
+                            segmentName: 'Name',
+                            internalID: 'Internal ID',
+                            scriptID: 'Script ID',
+                            customRecordScriptID: 'Transaction column ID',
+                            mapping: 'Displayed as',
+                        },
+                        removeTitle: 'Remove custom segment/record',
+                        removePrompt: 'Are you sure you want to remove this custom segment/record?',
+                        addForm: {
+                            customSegmentName: 'custom segment name',
+                            customRecordName: 'custom record name',
+                            segmentTitle: 'Custom segment',
+                            customSegmentAddTitle: 'Add custom segment',
+                            customRecordAddTitle: 'Add custom record',
+                            recordTitle: 'Custom record',
+                            segmentRecordType: 'Do you want to add a custom segment or a custom record?',
+                            customSegmentNameTitle: "What's the custom segment name?",
+                            customRecordNameTitle: "What's the custom record name?",
+                            customSegmentNameFooter: `You can find custom segment names in NetSuite under *Customizations > Links, Records & Fields > Custom Segments* page.\n\n_For more detailed instructions, [visit our help site](${CONST.NETSUITE_IMPORT.HELP_LINKS.CUSTOM_SEGMENTS})_.`,
+                            customRecordNameFooter: `You can find custom record names in NetSuite by entering the "Transaction Column Field" in global search.\n\n_For more detailed instructions, [visit our help site](${CONST.NETSUITE_IMPORT.HELP_LINKS.CUSTOM_SEGMENTS})_.`,
+                            customSegmentInternalIDTitle: "What's the internal ID?",
+                            customSegmentInternalIDFooter: `First, make sure you've enabled internal IDs in NetSuite under *Home > Set Preferences > Show Internal ID.*\n\nYou can find custom segment internal IDs in NetSuite under:\n\n1. *Customization > Lists, Records, & Fields > Custom Segments*.\n2. Click into a custom segment.\n3. Click the hyperlink next to *Custom Record Type*.\n4. Find the internal ID in the table at the bottom.\n\n_For more detailed instructions, [visit our help site](${CONST.NETSUITE_IMPORT.HELP_LINKS.CUSTOM_LISTS})_.`,
+                            customRecordInternalIDFooter: `You can find custom record internal IDs in NetSuite by following these steps:\n\n1. Enter "Transaction Line Fields" in gloabl search.\n2. Click into a customr record.\n3. Find the internal ID on the left-hand side.\n\n_For more detailed instructions, [visit our help site](${CONST.NETSUITE_IMPORT.HELP_LINKS.CUSTOM_SEGMENTS})_.`,
```

Comment:
```suggestion
                            customRecordInternalIDFooter: `You can find custom record internal IDs in NetSuite by following these steps:\n\n1. Enter "Transaction Line Fields" in global search.\n2. Click into a customer record.\n3. Find the internal ID on the left-hand side.\n\n_For more detailed instructions, [visit our help site](${CONST.NETSUITE_IMPORT.HELP_LINKS.CUSTOM_SEGMENTS})_.`,
```

Reply:
Good catch. I've fixed the typo and it's `custom`